simulator/config/two-sigma-settings.edn

```diff
@@ -0,0 +1,65 @@
+{:sim-db-uri "datomic:free://localhost:4334/cook-sim"
+ :cook-db-uri "datomic:free://localhost:4334/cook-jobs"
+ :cook-api-uri "http://localhost:12321"
+ :process-count 10
+ :sim-model {:label "Two Sigma's cluster usage"
```

Comment:
Can we scrub this of Two Sigma?

Just call it something like large-simulation. Change simulation running to video encoding (or something similar) and fcat to python users.